Clinical-Genomics/scout
# Store simple acmg

This PR adds a functionality or fixes a bug.

**How to test**:
1. Fetch the branch and install
1. Add a "simple" ACMG annotation, go back to variants list and check that it shows up(has to be P/LP). Now go back to variant page and unbox the annotation. See that it disappears.

**Expected outcome**:
This is described in issue 

**Review:**
